src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -711,11 +711,22 @@ object Parser {
         dereference |
         receiveExp |
         unfolding |
+        make |
+        newExp |
         len |
         cap |
         ghostUnaryExp |
         primaryExp
 
+    lazy val make : Parser[PMake] =
+      "make" ~> ("(" ~> typ ~ expSequence <~ ")") ^^ PMake
+
+    lazy val expSequence: Parser[Vector[PExpression]] =
+      ("," ~> rep1sep(expression, ",") <~ ",".?).? ^^ (opt => opt.getOrElse(Vector.empty))
```

Comment:
Is a comma at the end (e.g. `make([]int, 5,)`) just accepted by the Go compiler or is this part of the spec? I couldn't find anything about this in the section on `make` but the compiler indeed seems to be happy

Reply:
I did not find anything about this in the spec but, empirically, I have seen trailing commas working on function/method calls and some other places in *Go*. I found however [this stack overflow entry](https://stackoverflow.com/questions/29353429/golang-commas-with-methods-functions) that poses a question related to yours. Given that the compiler accepts calls like `make([]int, 5,)`, I did not bother to change the parser.

Reply:
Haha okay somehow reminds me of C++ compilers and undefined behavior ^^